.github/workflows/build-test.yml

```
@@ -114,3 +119,4 @@ jobs:
       ignore_errors: ${{ inputs.ignore_errors || false }}
       skip_list: ${{ inputs.skip_list }}
       run_name: ${{ inputs.run_name }}
+      enable_unskip: ${{ (github.event_name == 'workflow_dispatch' && inputs.enable_unskip) || false }}
```

Nit: i think this can be simpler:
```suggestion
      enable_unskip: ${{ inputs.enable_unskip || false }}
```